tslib/src/functions-ps-report/ps-report-4-writer/index.ts

```diff
@@ -38,8 +38,7 @@ async function bulkUpload (context: Context, incomingMessage: PsReportStagingCom
       await jobDataService.setJobComplete(incomingMessage.jobUuid,
         JobStatusCode.Failed, JSON.stringify(error))
     }
-    // Live debugging: 2024-05-31 for PS report. To be re-enabled.
-    // await service.cleanup(incomingMessage.filename)
+    await service.cleanup(incomingMessage.filename)
```

Comment:
let's leave the file intact until we are happy that everything is working correctly

Reply:
Done